src/plugins/kibana/public/settings/sections/status/index.js

```diff
@@ -5,6 +5,6 @@ define(function (require) {
     order: 3,
     name: 'status',
     display: 'Status',
-    url: '/status'
+    url: '../status'
```

Comment:
Rather than scatter path traversal logic all over whenever we need to create a url path in code, how about a utility function that takes in a path and applies the configured url base path?